astrobee/behaviors/inspection/src/inspection_node.cc

```diff
@@ -844,7 +851,11 @@ class InspectionNode : public ff_util::FreeFlyerNodelet {
     case STATE::INIT_INSPECTION:
       msg.fsm_state = "INIT_INSPECTION";                   break;
     case STATE::MOVING_TO_APPROACH_POSE:
-      msg.fsm_state = "MOVING_TO_APPROACH_POSE";           break;
+      msg.fsm_state = "MOVING_TO_APPROACH_POSE ";
+      msg.fsm_state += std::to_string(inspection_->GetCurrentCounter());
```

Comment:
```suggestion
      msg.fsm_state += std::to_string(inspection_->GetCurrentCounter() + 1);
```
I think this would be less confusing for most users using 1-based indexing. Reasoning:
- "MOVING_TO_APPROACH_POSE 56/56" seems correct at the end of the inspection.
- I guess with 1-based indexing you'd see the line number of the current pose in the inspection file, as shown in an editor.

Reply:
Ha never mind, you already added 1 earlier ;)